src/cap-description.adoc

```diff
@@ -145,8 +145,11 @@ permission.
 WARNING: #TODO: Confirm that we need a separate permissions format for 32-bit
 and 64-bit.#
 
-NOTE: Valid capabilities must not have the permissions field set to a reserved
-value according to xref:cap_perms_encoding32[xrefstyle=short] when XLENMAX=32.
+NOTE: When XLENMAX=32 there is a single reserved permission encoding (see
+xref:cap_perms_encoding32[xrefstyle=short]). It is not possible for a tagged
+capability to have this value since <<CANDPERM>> will never create it. It is
+possible for untagged capabilities to have it. <<CGETPERM>> will interpret it
+as if it were 0b0000 (no permissions).
```

Comment:
The way this is worded forbids an extension later assigning meaning to it (yes, this is non-normative text, but you still don't want it to be contradictory)

Reply:
Good point. Would it be ok if I change it to  `It is currently not possible...  <<CGETPERM>> will currently interpret it...  This may change in future.`.

Reply:
Maybe something like this still allows for future extensibility:

```
If the permission encoding has reserved bit patterns (e.g. the format for XLENMAX=32 with {cheri_base_ext_name}
xref:cap_perms_encoding32[xrefstyle=short]) then it is not possible for a tagged
capability to have this value since <<CANDPERM>> will never create it. However, it is
possible for untagged capabilities to have it in which case <<CGETPERM>> will interpret it
as if it were 0b0000 (no permissions). Future extensions may assign meaning to these bit patterns for which <<CANDPERM>> is allowed to report a non-zero value.
```

Reply:
Updated based on @arichardson's suggestion.